recipes/pyrepl/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyrepl" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "3bcbbf1e47d2b80819b805aceea16dc90832ba4800035d595fa3c4bf48fcd4d7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```

Comment:
Is `setuptools` needed at runtime? If not, would recommend dropping it and rebuilding the package with a new build number.